x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/__mocks__/view_index.mock.ts

```diff
@@ -5,25 +5,19 @@
  * 2.0.
  */
 
-import { ENTERPRISE_SEARCH_CONNECTOR_CRAWLER_SERVICE_TYPE } from '../../../../common/constants';
+import {ENTERPRISE_SEARCH_CONNECTOR_CRAWLER_SERVICE_TYPE} from '../../../../common/constants';
```

Comment:
Is it possible that your editors formatter conflicts with the prettier configuration we have?

Reply:
Yep, looks like it 😞